workloads/network-perf/README.md

```diff
@@ -1,211 +1,69 @@
-
-### ADDRESSPOOL
-Default: `addresspool-l2`  
-Used only when `SERVICETYPE` is `metallb`  
-To provide MetalLB addresspool for a service, this will be used as LoadBalancer network.  
-Mentioned addresspool should be pre-provisioned before execution of this script.  
+## Workload variables
+
+The run.sh script can be tweaked with the following environment variables
+
+| Variable                | Description              | Default |
+|-------------------------|--------------------------|---------|
+| **WORKLOAD**            | Networking workload, can be either pod2pod, pod2pod, hostnet or, smoke | smoke |   
+| **OPERATOR_REPO**       | Benchmark-operator repo                     | https://github.com/cloud-bulldozer/benchmark-operator.git |
+| **OPERATOR_BRANCH**     | Benchmark-operator branch                     | master      |
+| **ES_SERVER**           | Elasticsearch endpoint         | https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com:443 |
+| **METADATA_COLLECTION** | Enable metadata collection | true (If indexing is disabled metadata collection will be also disabled) |
+| **HOSTNETWORK**         | If enabled, will test the performance of the node the pod will run on | false |
+| **NETWORK_POLICY**      | If enabled, benchmark-operator will create a network policy to allow ingress trafic in uperf server pods | false |
+| **SERVICETYPE**         | To provide specifics about openshift service types, supported options `clusterip`, `nodeport`, `metallb`. `metallb` type requires manual installation of operators and configuration of BGPPeers as explained [here](https://github.com/cloud-bulldozer/benchmark-operator/blob/master/docs/uperf.md#advanced-service-types) | clusterip |
+| **ADDRESSPOOL**         | To provide MetalLB addresspool for a service, this will be used as LoadBalancer network. Mentioned addresspool should be pre-provisioned before execution of this script. | addresspool-l2 |
+| **SERVICE_ETP**         | To mention the type of `ExternalTrafficPolicy` of a service, supported option `Cluster` or `Local` | Cluster |
+| **SAMPLES**             | How many times to run the tests | 3 |
+| **MULTI_AZ**            | If true, uperf client and server pods will be colocated in different topology zones or AZs (`topology.kubernetes.io/zone` in k8s terminology). 2 worker nodes in differet topology zones are required to enable this flag.  If false, uperf client and server pods will be colocated in the same topology zone | true |
+| **PAIRS**               | List with the number of pairs the test will be triggered | 1 2 4 |
+| **TEST_TIMEOUT**        | Benchmark timeout, in seconds | 7200 (2 hours) |
+| **TEST_CLEANUP**        | Remove benchmark CR at the end | true | 
+| **CERBERUS_URL**        | URL to check the health of the cluster using Cerberus | "" |
```

Comment:
Can we remove Cerberus since we haven't ever really used it?

Reply:
I was tempted, but... it's not out of scope of this PR? :)

Reply:
I mean if you are doing a full re-write I'd say its in scope but i'll let the others chime in as well.

Reply:
Added a new commit to remove it ;)